### PR TITLE
[wasm] Enable SIMD.

### PR DESCRIPTION
**Pull request by annxingyuan**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/3355)

### CODE REVIEW

File: tfjs-backend-wasm/README.md [hunk 1]
```diff
@@ -108,13 +108,13 @@ JS backend, and ~5.3-7.7X slower than the WebGL backend.
...
-| iPhone XS                | 140   | 18.1  | 426.4    |
-| Pixel 3                  | 266.2 | 77.3  | 2345.2   |
-| Desktop Linux            | 91.5  | 17.1  | 1049     |
-| Macbook Pro              | 98.4  | 19.6  | 893.5    |
+| MobileNet inference (ms) | WASM  | WebGL | Plain JS | WASM + SIMD |
+|--------------------------|-------|-------|----------|-------------|
+| iPhone X                 | 147.1 | 20.3  | 941.3    | N/A         |
+| iPhone XS                | 140   | 18.1  | 426.4    | N/A         |
+| Pixel 3                  | 266.2 | 77.3  | 2345.2   | N/A         |
+| Desktop Linux            | 91.5  | 17.1  | 1049     | N/A         |
```
Comment (Maratyszcza) [inline tfjs-backend-wasm/README.md:116]: You may benchmark on desktop Linux with Chrome Dev or Firefox Nightly

File: tfjs-backend-wasm/README.md [hunk 2]
```diff
@@ -108,13 +108,13 @@ JS backend, and ~5.3-7.7X slower than the WebGL backend.
...
-| iPhone X                 | 147.1 | 20.3  | 941.3    |
-| iPhone XS                | 140   | 18.1  | 426.4    |
-| Pixel 3                  | 266.2 | 77.3  | 2345.2   |
-| Desktop Linux            | 91.5  | 17.1  | 1049     |
-| Macbook Pro              | 98.4  | 19.6  | 893.5    |
+| MobileNet inference (ms) | WASM  | WebGL | Plain JS | WASM + SIMD |
+|--------------------------|-------|-------|----------|-------------|
+| iPhone X                 | 147.1 | 20.3  | 941.3    | N/A         |
+| iPhone XS                | 140   | 18.1  | 426.4    | N/A         |
+| Pixel 3                  | 266.2 | 77.3  | 2345.2   | N/A         |
```
Comment (Maratyszcza) [inline tfjs-backend-wasm/README.md:115]: Chrome on Android supports SIMD

File: tfjs-backend-wasm/wasm-out/tfjs-backend-wasm-simd.d.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
```
Comment (Maratyszcza) [inline tfjs-backend-wasm/wasm-out/tfjs-backend-wasm-simd.d.ts:3]: 2020 :)

File: tfjs-backend-wasm/README.md (hunk 2 above)
Comment (annxingyuan) [reply tfjs-backend-wasm/README.md:115]: Hmm, for some reason the support test I'm using (https://github.com/GoogleChromeLabs/wasm-feature-detect) returns false on Chrome Android, even though in chrome://flags it says it's supported.

Comment (annxingyuan) [reply tfjs-backend-wasm/README.md:115]: Ah turns out the feature test only works in Chrome 84+, and I was using Chrome 80. Will work on getting these benchmarks :)

**Pull request merged**